buildSrc/src/main/java/Dependencies.kt

```diff
@@ -11,7 +11,7 @@ object Versions {
     const val dataBinding = "3.2.0-alpha11"
     const val kotlinVersion = "1.3.21"
     const val supportLib = "1.0.0"
-    const val designSupportLib = "1.0.0"
+    const val designSupportLib = "1.1.0-rc02"
```

Comment:
Does this dependency need to be upgraded in order to use ViewPager 2? If not, please remove this change.

Reply:
It's needed because previous version had bugs and some features were added later.

Reply:
By "some features", do you mean features that are required for ViewPager2 to work?